ruby/arguments.rb

```diff
@@ -0,0 +1,10 @@
+"""Script to read an argument and print as lowercase in stdout.
+Note that command line arguments are not passed from stdin"""
+
+if ARGV.length != 1
+    puts "Make sure to enter an argument!"
+    exit(1)
+end
+
+argument = ARGV[0].downcase
```

Comment:
Depending on how terse we'd want to make it, this could be `puts ARGV[0].downcase`.

If there aren't arguments given, `ARGV[0]` will return `nil` (like Python's `None`) and then calling `.downcase` on it will raise a `NoMethodException` (or something to that effect), so we'd get an error message (granted a less helpful one) if you don't pass an argument.

Reply:
Are you saying we should skip the arg length checks? I'm fine with that as it'll simplify the scripts

Reply:
I'm on the fence, but I think it's safe to skip the check once we get the test script (e.g. `ruby/arguments.rb`) working. If we refactor the `test_suite.py` or the Dockerfile and this test breaks, we probably won't have a hard time figuring out why it's broken.